.github/ISSUE_TEMPLATE/inception-request.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Inception request
+about: Suggest an idea for OFN's Inception process
+title: "[Inception Request: Name your problem statement here]"
+labels: 'Blocked:Design Input needed, Blocked: Tech Input needed, Blocked:Product
+  input neeed'
+assignees: ''
+
+---
+
+**Is your request related to a problem? Please describe in as much specific detail as possible.**
+
+*e.g. A clear and concise description of what the problem is. Ex. We want to find out how to include a process where shoppers can facilitate a better group buying process in OFN [...]*
+
+
+
+**(Essential information) What areas of the OFN product or experience are 'affected' by this project, feature or idea?**
+
+*e.g. 
+- The back office product list and inventory.
+- The shopper experience when buying in bulk.*
+
+
+
+**(Essential information) Who are the personas/stakeholders/users that need to be considered as 'key' within this project, feature or idea?**
+
+*e.g. 
+- A super admin type user.
+- A new shopper.
+- A new producer/farmer in rural India.*
+*You can find an ever evolving list of Personas for OFN [here](https://www.notion.so/openfoodnetwork/Audiences-and-personas-cd7a2d86c6fc4656b4567acf0c94fe91)*
+
+
```

Comment:
** Acceptance Criteria**: What does the solution need to be successful?
_This does not have to be complete or on a very detailed/ technical yet, but try to give a high level idea 

*e.g.
- Shopper is enabled shopper to do x
- Shopper cannot do x without y
- Admin needs to agree on x

Reply:
Same here, kill the Admin example ;)

Reply:
Done